Remotion/Web/Test.Shared/Web.Test.Shared.csproj

```diff
@@ -0,0 +1,30 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <ProjectType>WebLibrary</ProjectType>
+  </PropertyGroup>
+  <Import Project="..\..\..\Build\Shared.build.props" />
+  <PropertyGroup>
+    <AssemblyName>Remotion.Web.Test.Shared</AssemblyName>
+    <RootNamespace>$(AssemblyName)</RootNamespace>
+    <NoWarn>$(NoWarn);0419</NoWarn>
+    <Nullable>disable</Nullable>
+  </PropertyGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == 'net461'">
+    <Reference Include="System.Configuration" />
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\Core\Core\Core.Core.csproj" />
+    <ProjectReference Include="..\..\Data\Core\Data.Core.csproj" />
+    <ProjectReference Include="..\..\Development\Web\Development.Web.csproj" />
+    <ProjectReference Include="..\..\Globalization\Core\Globalization.Core.csproj" />
+    <ProjectReference Include="..\..\Security\Core\Security.Core.csproj" />
+    <ProjectReference Include="..\Core\Web.Core.csproj" />
+  </ItemGroup>
+  <ItemGroup>
+    <Content Include="Web.config">
+      <SubType>Designer</SubType>
+    </Content>
```

Comment:
I thought that a Web.config with the page imports would allow the .aspx files to correctly detect the controls in the remotion namespace. But it seems like this is not the case anyway. How should we handle this problem? Import the namespace in each and every .as?x file?

Reply:
Hmm...I'm not sure I understand which problem you're referring to. I can think of three possible problem scenarios with different scariness-level. Let's talk tomorrow.